### DISCUSSION TITLE
跑分专帖的视频解码部分疑似存在误差

### DISCUSSION THREAD

**Discussion by ghost** [hooke007/mpv_PlayKit#263] [mpv 通用]

**Comment by hooke007** [comment]
不太明白你的意思，mpv要完成完整的渲染流程，这和ffmpeg有什么关系呢

**Reply by hooke007** [reply]
> 产生插值帧

没有使用插值

这就是在解除垂直同步条件下测的解码能力，你是不是理解有误？

**Reply by hooke007** [reply]
这个测试方式源于开发者hassn。我追加了更多条件去测试不同场景。
该方式测得的结果也基本对应不同场景高负载下的掉帧率排名。

至于你的ffmpeg方法是不是有问题，我不知道。

**Reply by hooke007** [reply]
我知道它原始用于测试渲染，在mpv中解码不可能单独存在
正如前面所说，不同的场景下不同的解码和跑分排名是匹配的

**Reply by hooke007** [reply]
我看懂你前面说的了，就是估计帧率比源总帧数还高。
是的，因为那个estimated属性不可靠，总体而言看的还是排名和差距 https://mpv.io/manual/master/#command-interface-estimated-frame-number

**Reply by hooke007** [reply]
> display-resample

这是额外的问题。极大受显示器的刷新率影响 https://github.com/mpv-player/mpv/issues/11122

**Comment by hooke007** [comment]
`display-*` 会触发强制重采样的问题，，不过我也找不到更好的属性去反应实际数值了，，要不然就只能自己恰秒表了（

**Reply by hooke007** [reply]
确实，，而且居然更快了（动手掐表计算
